Makefile

```diff
@@ -237,7 +237,7 @@ test-e2e: docker
 	@echo ">> cleaning docker environment."
 	@docker system prune -f --volumes
 	@echo ">> cleaning e2e test garbage."
-	@rm -rf ./test/e2e/e2e_integration_test*
+	@rm -rf ./test/e2e/e2e_*
```

Comment:
:smile:

Reply:
:+1: this is good, I have a bunch of leftovers `e2e_12345` and so on :smile:

Reply:
Yeah, unfortunately this was forgotten during the e2e migration, should be good from now on :blush: